conventional_commits/action.yml

```diff
@@ -0,0 +1,89 @@
+name: Conventional commits
+description:
+  enforces conventional commit style in pull requests with Ory-specific defaults
+
+inputs:
+  GITHUB_TOKEN:
+    description: the GitHub token to use
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - id: load_config
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const CONFIG_PATH = ".github/conventional_commits.json"
```

Comment:
Can we rather make the scripts one JS file to have proper IDE support, testablility, ...? This seems super bad for maintainability.
To avoid having to use `actions/checkout`, we can just fetch the file from a github raw URL and execute it.